### ISSUE TITLE
Facilitate the creation of correct masks, e.g. with repeater GCPs

### ISSUE THREAD

**Issue by mclaeysb** [allmaps/allmaps#122]
Warn and/or fix incorrect masks when loading them. 

Create a function that checks for:
- repeated gcp's => consider to use the conformPolygon() function in the @allmaps/stdlib to filter out duplicated points (one of the most common errors)!
- self-intersections => consider to use [this module](https://github.com/mclaeysb/geojson-polygon-self-intersections)
- collinearity
- winding order => see turf rewind and [placemark docs](https://github.com/placemark/docs/blob/main/docs/winding-order.md) (also used in @allmaps/latest)
- less then 2 gpc's

Later this could be extended with warnings wrt the subjective quality of the gpc's.

This function could live in its own module. It could be called in the editor, renderer, or annotations package (using an option).

**Comment by sammeltassen** [comment]
@mclaeysb Is this now part of @allmaps/analyze?

**Comment by mclaeysb** [comment]
Yes, these checks are part of @allmaps/analyze now! But this module is not used yet in Editor or Viewer.

**Comment by sammeltassen** [comment]
Thanks, I'll leave it open for now
